cmd/restic/cmd_mount.go

```diff
@@ -90,10 +90,13 @@ func mount(opts MountOptions, gopts GlobalOptions, mountpoint string) error {
 		return err
 	}
 
-	lock, err := lockRepo(repo)
-	defer unlockRepo(lock)
-	if err != nil {
-		return err
+	if !gopts.NoLock {
+		Verbosef("create lock for repository\n")
```

Comment:
Instead of printing out in info when the repo is locked, I would print out a warning when the repo is *not* locked.

Reply:
Commit updated. Thanks.